code/lib/codemod/package.json

```diff
@@ -46,7 +46,7 @@
     "globby": "^11.0.2",
     "jscodeshift": "^0.13.1",
     "lodash": "^4.17.21",
-    "prettier": ">=2.2.1 <=2.3.0",
+    "prettier": ">=2.2.1 <=3.0.0",
```

Comment:
Why not simply upgrade to the latest?

Is 3.0.0 a no-go? if so, why?

Reply:
3.0 is making breaking changes, and is only in alpha stage right now.  I thought maybe it's worth waiting to see how it shakes out for a bit.  I'd have to look exactly how we use prettier to see if we're impacted.